### PR TITLE
uploads-pop-up-cy-253

### PR DESCRIPTION
**Pull request by cris22380**

### CODE REVIEW

File: src/components/uploads-pop-up/styles/SUploadsPopup.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import styled, { css } from "styled-components";
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../constants/styles/mediaquerys";
+
+interface ISUploadsPopup {
+}
+
+const Bronze = css<ISUploadsPopup>`
+display: flex;
```
Comment (diegomart2000) [inline src/components/uploads-pop-up/styles/SUploadsPopup.tsx]: @cris22380 fix this format error.

File: src/components/uploads-pop-up/UploadsPopup.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const UploadsPopUp: FC<IUploadsPopUp> = ({children, poupTitle, onClose}) => (
+  <SUploadsPopup>
+    <SUploadsPopupHeader>
+      <PrimaryModalHeader withCloseIcon={true} breadcrumbs={[{title: poupTitle, icon: <IconUpload size={24}/> }]} onClose={onClose} >
+        <BadgeWrapper>
+          <Badge type="regular" value="1" size="small" position={EBadgePosition.Center}/>
+        </BadgeWrapper>
+        <ButtonsWrapper>
+          <FlatActionButton type="grey">Clear</FlatActionButton>
```
Comment (diegomart2000) [inline src/components/uploads-pop-up/UploadsPopup.tsx]: @cris22380 clear button needs a callback.

File: src/components/uploads-pop-up/UploadsPopup.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          <IconWrapper onClick={onClose}><IconClose size={24}/></IconWrapper>
+        </ButtonsWrapper>
+      </PrimaryModalHeader>
+    </SUploadsPopupHeader>
+
+    {children}
```
Comment (diegomart2000) [inline src/components/uploads-pop-up/UploadsPopup.tsx:31]: @cris22380 missing see all assets button.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]
Screen Shot 2022-01-12 at 9 13 28 AM

Icon is wrong, missing container styles.

**Comment by cris22380** [comment]
- From Storybook

Screen Shot 2022-01-12 at 10 34 09

- From Storybook

Screen Shot 2022-01-12 at 10 34 26

- From figma

Screen Shot 2022-01-12 at 10 36 19

- From figma

Screen Shot 2022-01-12 at 10 37 52

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**